en/docs/services/create-your-first-service.md

```diff
@@ -100,23 +101,23 @@ To create the service, follow the procedure below:
     
 9. To add a statement of the `Respond` type so that the result of the above processing is returned as a response, click the last **+** icon in your low-code diagram.
 
-    ![Add Respond Statement](../assets/img/services/add-respond-statement.png)
+    ![Add Respond Statement](../assets/img/services/add-respond-statement.png){.cInlineImage-half}
```

Comment:
This image is too lengthy too. can we follow the same as suggested above?

Reply:
Image size is not visible in github view mode. This shows the original image size.